iina/Base.lproj/Localizable.strings

```diff
@@ -11,78 +11,78 @@
+"osd.chapter" = "Chapitre: %@";
+"osd.subtitle_scale" = "Échelle de Sous-Titres: %.2fx";
+"osd.add_to_playlist" = "%i Fichiers ajoutés a la liste de lecture";
+"osd.clear_playlist" = "Liste de lecture Frayée";
+"osd.video_eq.contrast" = "Contraste: %i";
 "osd.video_eq.gamma" = "Gamma: %i";
-"osd.video_eq.hue" = "Hue: %i";
+"osd.video_eq.hue" = "Teint: %i";
 "osd.video_eq.saturation" = "Saturation: %i";
-"osd.video_eq.brightness" = "Brightness: %i";
+"osd.video_eq.brightness" = "Luminosité: %i";
 
-"update.title_update_found" = "Update Found";
-"update.title_no_update" = "No Update";
-"update.title_error" = "Error";
-"update.update_found" = "New version detected! IINA is still in alpha, so you are recommended to stay with the latest version. Please download from website or github:";
-"update.no_update" = "You are running the latest version.";
-"update.check_failed" = "Failed to check updates. You are probably running a too old version. Please check the website or github.";
-"update.new_version" = "New version:";
-"update.dl_from_website" = "Go to Website";
-"update.dl_from_github" = "Go to Github";
+"update.title_update_found" = "Mise à jour Trouvée";
+"update.title_no_update" = "Aucune Mise à jour Trouvée";
+"update.title_error" = "Érreur";
+"update.update_found" = "Mise à jour trouvée! IINA est toujours en alpha, alors il est recommendé d'avoir la plus récente version. Les fichiers sont disponible sur le site web IINA ou sur GitHub:";
+"update.no_update" = "Vous avez la version la plus récente.";
+"update.check_failed" = "Echec de trouvaille de mise à jour. La version courrante est trop vieille. Les plus récent fichiers sont disponible sur le site web IINA ou sur GitHub.";
+"update.new_version" = "Nouvelle version:";
+"update.dl_from_website" = "Ouverture du site Web";
+"update.dl_from_github" = "Ouverture de la page Github";
 
-"preference.title" = "Preference";
-"preference.general" = "General";
-"preference.ui" = "UI";
+"preference.title" = "Préferences";
+"preference.general" = "Général";
+"preference.ui" = "Interface d'Utilisateur";
 "preference.codec" = "Codec";
-"preference.subtitle" = "Subtitle";
-"preference.network" = "Network";
-"preference.control" = "Control";
-"preference.keybindings" = "Keybindings";
-"preference.advanced" = "Advanced";
-"preference.scripts" = "Scripts";
+"preference.subtitle" = "Sous-Titres";
+"preference.network" = "Réseau";
+"preference.control" = "Controle";
+"preference.keybindings" = "Raccourcis";
+"preference.advanced" = "Paramètres Avancés";
+"preference.scripts" = "Scriptes";
 
 "menu.volume"= "Volume: %d";
-"menu.audio_delay" = "Audio Delay: %.2fs";
-"menu.sub_delay" = "Subtitie Delay: %.2fs";
-"menu.fullscreen" = "Full Screen";
-"menu.exit_fullscreen" = "Exit Full Screen";
-"menu.pause" = "Pause";
-"menu.resume" = "Resume";
+"menu.audio_delay" = "Délai Audio: %.2fs";
+"menu.sub_delay" = "Délai de Sous-Titres: %.2fs";
+"menu.fullscreen" = "Écran Plein";
+"menu.exit_fullscreen" = "Fermer l'Écran Plein";
```

Comment:
in Apple apps, it's often "Quitter le mode plein écran"
![capture d ecran 2017-01-28 15 16 07](https://cloud.githubusercontent.com/assets/1141722/22397198/b862ee7a-e56c-11e6-8b50-13b8e0b2209f.png)